tests/installer/installer_test.go

```diff
@@ -204,7 +204,7 @@ var _ = Describe("cOS Installer tests", func() {
 				Expect(out).To(ContainSubstring("Preparing recovery.."))
 				Expect(out).To(ContainSubstring("Preparing passive boot.."))
 				Expect(out).To(ContainSubstring("Formatting drives.."))
-				Expect(out).To(ContainSubstring("fatal error: 1 error occurred:"))
+				Expect(out).To(ContainSubstring("Error: 1 error occurred:"))
```

Comment:
cant we just remove the error part and check for "x error occurred" ? In case we change logger output down the line

Reply:
Done!